src/App.jsx

```diff
@@ -0,0 +1,45 @@
+import {useState, useEffect} from "react";
+import './App.css';
+import BlogList from "./Dashboard/BlogList";
+import { TextBox } from "./Components/TextBox";
+
+function App() {
+  const [data, setData] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [error, setError] = useState('');
+  
+  const handleDelete = (idx) => {
+    const res = data.filter((item)=> item?.id !== idx);
+    setData(res);
+  }
+
+  useEffect(()=> {
+    fetch("https://jsonplaceholder.typicode.com/posts")
```

Comment:
you have to write own custom hook useFectch

Reply:
Need to create seperate folder for fetching the data?